frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -111,12 +123,59 @@ export function Surveys(): JSX.Element {
                     { key: SurveysTabs.Archived, label: 'Archived' },
                     showLinkedHogFunctions ? { key: SurveysTabs.Notifications, label: 'Notifications' } : null,
                     { key: SurveysTabs.History, label: 'History' },
+                    surveysConfigAvailable ? { key: SurveysTabs.Settings, label: 'Settings' } : null,
                 ]}
             />
-
-            {tab === SurveysTabs.History ? (
-                <ActivityLog scope={ActivityScope.SURVEY} />
-            ) : tab === SurveysTabs.Notifications ? (
+            {tab === SurveysTabs.Settings && (
+                <>
+                    <div className="flex items-center mb-2 gap-2">
+                        <div className="flex-1" />
+                        <LemonButton
+                            type="primary"
+                            onClick={() => {
+                                updateCurrentTeam({
+                                    survey_config: {
+                                        ...currentTeam?.survey_config,
+                                        appearance: {
+                                            ...currentTeam?.survey_config?.appearance,
+                                            ...editableSurveyConfig,
+                                        },
+                                    },
+                                })
+                            }}
+                        >
+                            Save settings
+                        </LemonButton>
+                    </div>
+                    <div className="flex flex-col overflow-hidden flex-1">
+                        <LemonCollapse
+                            activeKey="Appearance"
+                            panels={[
+                                {
+                                    key: 'Appearance',
+                                    header: 'Appearance',
+                                    content: (
+                                        <Customization
+                                            key="survey-settings-customization"
+                                            appearance={editableSurveyConfig}
+                                            hasBranchingLogic={false}
+                                            customizeRatingButtons={true}
+                                            customizePlaceholderText={true}
+                                            onAppearanceChange={(appearance) => {
+                                                setEditableSurveyConfig({
+                                                    ...editableSurveyConfig,
+                                                    ...appearance,
+                                                })
+                                            }}
+                                        />
+                                    ),
+                                },
+                            ]}
+                        />
```

Comment:
IMO don't use a LemonCollapse element here; I don't think that UX makes sense when we don't have a list of collapsible elements (i.e., it makes sense in the survey editor, but not in this context where we only have one element that doesn't collapse).

Reply:
Done, removed this now